frontend/src/components/Forms/ShoutoutsPage/ShoutoutCard.tsx

```diff
@@ -24,7 +43,20 @@ const ShoutoutCard = (props: {
         <ShoutoutDeleteModal uuid={shoutout.uuid} setGivenShoutouts={setGivenShoutouts} />
       </Card.Group>
       <Card.Content description={shoutout.message} />
+
+      {isLoading ? (
+        <Loader active inline />
+      ) : (
+        image && (
+          <Card.Content>
+            <div className={styles.imageContainer}>
+              <Image src={image} size="small" />
```

Comment:
The Image component should have an alt attribute for accessibility purposes since it seems that the image is being dynamically loaded!